c10/util/Float8_e5m2fnuz-inl.h

```diff
@@ -270,6 +270,11 @@ class numeric_limits<c10::Float8_e5m2fnuz> {
   static constexpr c10::Float8_e5m2fnuz infinity() {
     return c10::Float8_e5m2fnuz(0x80, c10::Float8_e5m2fnuz::from_bits());
   }
+  // TODO(future): we are mapping neg_zero to both inf and NaN, this is
```

Comment:
FWIW I think this is fine, but there may be additional downstream edge cases that will need to be fixed.